pkg/datapath/iptables/iptables.go

```diff
@@ -125,6 +125,21 @@ func getFeedRule(name, args string) []string {
 	return append(argsList, ruleTail...)
 }
 
+// skipPodToPodConntrack returns true if it's possible to install iptables `-j
+// NOTRACK` rules to skip tracking pod to pod traffic.
+//
+// These rules will be installed only when Cilium is running in strict KPR mode
+// (as otherwise conntrack would be required for NAT operations) and in direct
+// routing mode (as in tunneling mode the encapsulated traffic is already
+// skipping netfilter conntrack).
+func skipPodToPodConntrack(ipv6 bool) bool {
+	return !ipv6 && option.Config.EnableIPv4 &&
```

Comment:
Why not IPv6?

Reply:
Because the native routing CIDR (which is used to select all the pod traffic) can only be an IPv4 CIDR.
The `ipv6` check is needed as this function is also [called from](https://github.com/cilium/cilium/pull/15264/files#diff-465a6b3f4ab9e1e94fe39e5aefe81c41eaa1dc99c66500161072e564029c2f51R756) `InstallEndpointNoTrackRules`, which is a different thing (it installs NOCT rules for specific endpoints rather than a global rule to match all pod to pod traffic).

So, if we are trying to install a per endpoint NOCT rule:
* if the rule is for IPv4 and the pod to pod NOCT option is enabled, we skip installing more specific per endpoint rules
* otherwise, for IPv6, we install it

Reply:
Ok, thanks for the detailed explanation. Would that be worth mentioning in the comments above the function?